.github/CONTRIBUTING.md

```diff
@@ -4,12 +4,14 @@ Anyone is welcome to make BeEF better!
 
 Below are the steps needed to add code to BeEF on Github to the master branch:
 
-1. Clone the repository and create a new branch
+1. Fork the repository and create a new branch
 
 2. Write and commit your new code to that branch
 
 3. Run the existing tests to make sure they pass
 
+4. Make a pull request from the forked branch to BeEF master
```

Comment:
@jcrew99 - point 7. in that list already details creating a pull request. Maybe add the 'from your fork's branch to BeEF master' to that point, otherwise users may get confused.

Reply:
For reference, here's some notes I prepared earlier for a different project.

```
# Checkout the master branch
git checkout master

# Create a new branch for your feature
git checkout -b my-cool-new-feature

# Add your new files
git add modules/my-cool-new-module

# Commit your changes with a relevant message
git commit

# Push your changes to GitHub
git push origin my-cool-new-feature

# Now browse to the following URL and create your pull request!
# - https://github.com/beefproject/beef/pulls
```